independent-publisher-connectors/Proposal - 24 Pull Request (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "24 pull request",
+    "description": "connects to '24 pull request' api for open source projects contibution",
```

Comment:
Hi @Bkaraba,
Thank you so much for submitting your connector proposal.
I wanted to bring your attention to some points you may need to fix.
Please remove the API words from the description. Please follow [these requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) for the description.